src/main/resources/client.properties

```diff
@@ -0,0 +1 @@
+ilp.grpc.url=https://localhost:6565
```

Comment:
Just for my knowledge, can you expand on what's happening here. 

Generally when I consume libraries, I assume that I can configure them through injecting configuration in the constructor. This, by contrast, is using a static file to load properties off of and assigning a default. 

Is this a Spring construct? A Java construct? Just something that the ILP team likes? 

Should we support both of these things?

Reply:
When I wrote this, I assumed from `XpringClient` that we did not want a library consumer to have to configure the GRPC URL.  Instead of hardcoding a gRPC URL in `DefaultIlpClient`, I thought it would be better to make it configurable via a `.properties` file.

Spring has much more support for this, but in plain Java, this is the equivalent of a `.env` file.  Just more annoying to deal with `.properties` files in Java...

Question is, assuming we will now let the consumer configure the grpc url in the `IlpClient` constructor, do we still want a no-arg constructor?  If so, should we just hard-code a default grpc url, or should we keep this pattern of `.properties` files?

Reply:
@keefertaylor see https://github.com/xpring-eng/xpring4j/pull/103#discussion_r384779408